linuxamd64.Dockerfile

```diff
@@ -53,6 +53,11 @@ COPY --from=builder /go/bin/lncli /bin/
 COPY --from=builder /go/bin/lnd /bin/
 
 COPY docker-entrypoint.sh /docker-entrypoint.sh
+
+# Copy script for automatic init and unlock of lnd, need jq for parsing JSON and curl for LND Rest
+RUN apk --no-cache add jq curl
```

Comment:
you should add these to the initial `apk` call at line 27

Reply:
I like it here... so it's obvious these are the dependencies of this script.